### DISCUSSION TITLE
Xamarin + sdb samples

### DISCUSSION THREAD

**Discussion by jojojames** [mono/sdb#77] [Questions] [answered]
Would it be possible to show how to debug xamarin apps with sdb?

Noticed this was Xamarin + sdb https://github.com/mono/sdb/issues/8

Building an iOS or Android app should give relevant .exe and .mdb files but I'm not sure how to connect the dots to get it working end to end.

Thanks!

**Comment by alexrp** [comment, marked as answer]
Yes, it is possible. I haven't had the time to document the process properly in SDB's README, but you can follow these instructions (from the old Xamarin.Android internal README): https://gist.github.com/alexrp/18de95b295498ccec540a37e69374536

I don't work on Xamarin.iOS, so I am not sure what you would need to do there to get SDB working with it. It should certainly be possible, though - SDB and Xamarin Studio use the same underlying method for connecting.

**Comment by jojojames** [comment]
Thanks! I'll put something here if I find out how to do do it on iOS.
